### PR TITLE
Remove dynamic opacity in the Classic style 

### PR DESCRIPTION
**Pull request by jsanz**
Addresses #11 for the osm-bright style. I replaced dynamic opacity by fixed values, also adding zoom-scaled colors to adapt to this removal for `landcover-wood`.

> **Note**
> I need to figure out a way to run this comparison against `master` instead of our production styles.

Layers tweaked:

* `aeroway-area`
* `building-top`
* `landcover-glacier`
* `landcover-wood`
* `landcover-glass`
* `landcover-ice-shelf`
* `landcover-grass-park`

As before, `style.json` is generated automatically by running the `make` target and only the files in `layers` folder need review.

### Preview and some screenshots

Using Stamen's maperture tool: [`master` versus this PR style](https://stamen.github.io/maperture/#map=7.63/40.3618/-3.7542/0/0&showCollisions=false&showBoundaries=false&showDiff=false&viewMode=mirror&maps=%5B%7B%22name%22%3A%22Bright%22%2C%22type%22%3A%22mapbox-gl%22%2C%22renderer%22%3A%22mapbox-gl%22%2C%22url%22%3A%22https%3A%2F%2Fraw.githubusercontent.com%2Felastic%2Fosm-bright-gl-style%2Fmaster%2Fstyle.json%22%2C%22id%22%3A%22bright%22%2C%22index%22%3A0%7D%2C%7B%22name%22%3A%22Bright%22%2C%22type%22%3A%22mapbox-gl%22%2C%22renderer%22%3A%22mapbox-gl%22%2C%22url%22%3A%22https%3A%2F%2Fraw.githubusercontent.com%2Felastic%2Fosm-bright-gl-style%2F08bb34df54f8121140bd7273a8d15bd7de30573e%2Fstyle.json%22%2C%22id%22%3A%22bright%22%2C%22index%22%3A1%7D%5D)

image

image

image

### CODE REVIEW

File: layers/landcover/style.json [hunk 1]
```diff
@@ -51,20 +39,13 @@
         "visibility": "visible"
       },
       "paint": {
-        "fill-color": "#6a4",
-        "fill-opacity": {
-          "base": 0.1,
+        "fill-color": {
```
Comment (nickpeihl) [inline layers/landcover/style.json:42]: The fill color is very slightly darker now. But I think this is fine.

File: layers/landcover/style.json [hunk 2]
```diff
@@ -18,19 +18,7 @@
...
-              0,
-              0.9
-            ],
-            [
-              10,
-              0.3
-            ]
-          ]
-        }
+        "fill-opacity": 0.9
```
Comment (nickpeihl) [inline layers/landcover/style.json:21]: Do we want all styles to be fully opaque (e.g. 1)? Probably it makes little difference either way. 🤷

File: layers/landcover/style.json [hunk 3]
```diff
@@ -51,20 +39,13 @@
...
-            ],
-            [
-              9,
-              0.1
-            ]
+            [7, "rgba(218, 234, 214, 1)"],
+            [12, "rgba(199, 218, 199, 1)"]
           ]
         },
+        "fill-opacity": 0.5,
```
Comment (nickpeihl) [inline layers/landcover/style.json:48]: This opacity will be overwritten by Kibana, right? Does that cause issues when Kibana layer opacity is set to 100%?

Maybe I'm not understanding the root issue correctly?

Comment (jsanz) [reply layers/landcover/style.json:48]: The issue is only with dynamic opacity. With fixed opacity as we are setting here, Kibana will multiply this value by the opacity set for the whole map. That process is carried out here: 

https://github.com/elastic/kibana/blob/main/x-pack/plugins/maps/public/classes/layers/ems_vector_tile_layer/ems_vector_tile_layer.tsx#L413

My first attempt was to modify that `_setOpacityForType` method to process an opacity object by taking the `base` property but then I decided that it was a better solution to fix this directly on the styles.

File: layers/landcover/style.json (hunk 1 above)
Comment (jsanz) [reply layers/landcover/style.json:42]: Yeah, I was burning my eyes trying to figure out a tone close to the original enough at the different zoom levels and stopped at this point. It is a slight change, but if someone other than automation detects this I will send them my most sincere congratulations :sweat_smile:

**Review by nickpeihl — APPROVED** [review]
lgtm!

**Pull request merged**